### PR TITLE
Lint fixes

### PR DESCRIPTION
**Pull request by mohe2015**
This also improves continuous integration by linting and checking that the packaged version matches the committed dist file.

Example of the packaging test: https://github.com/mohe2015/annotations-action/actions/runs/451562023

### CODE REVIEW

**Comment by mohe2015** [comment]
Rebased to main.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -1,12 +1,19 @@
...
+    - uses: actions/checkout@v2
     - run: npm ci
+    - name: Testing / Linting
+      run: npm run test
+    - name: Package
+      run: npm run package
+    - name: "Check that packaging didn't create any changes"
+      run: '[ -z "$(git status --porcelain)" ];'
     - name: Annotate
+      if: ${{ github.event_name == 'push' }}
```
Comment (Mogztter) [inline .github/workflows/test.yml:16]: (nitpick): not 100% sure but I think it can be simplified

```suggestion
      if: github.event_name == 'push'
```

**Comment by Mogztter** [comment]
Looks good :ok_hand: 

@mohe2015 I left a comment, otherwise :+1:

**Comment by mohe2015** [comment]
@Mogztter Done. Also your suggested change seems to work, see https://github.com/mohe2015/annotations-action/tree/lint-fixes. I didn't keep co-authorship to keep the commits clean but I assume this is fine for such a simple change.

**Comment by Mogztter** [comment]
> @Mogztter Done. Also your suggested change seems to work, see https://github.com/mohe2015/annotations-action/tree/lint-fixes. I didn't keep co-authorship to keep the commits clean but I assume this is fine for such a simple change.

No worries :wink:

**Review by Mogztter — APPROVED** [review]
:+1:

**Pull request merged**